base/lthooks.dtx

```diff
@@ -2766,11 +2762,17 @@
 % \subsection{Adding or removing hook code}
 %
 % \begin{macro}{\hook_gput_code:nnn}
-% \begin{macro}{\@@_gput_code:nnn,\@@_gput_code:nxv,\@@_hook_gput_code_do:nnn}
+% \begin{macro}{\@@_gput_code:nnn,\@@_hook_gput_code_do:nnn}
+%
+%    \begin{macrocode}
+%<latexrelease>\IncludeInRelease{2021/11/15}%
+%<latexrelease>                 {\hook_gput_code:nnn}{Do~not~queue~removals}
```

Comment:
not sure we should really do a rollback or consider that a correction of the initial code that should have been this way day one. It complicates maintenance and I doubt that it has been used with the earlier meaning and if so a rollback would only be of limited use even then. Doing it wouldn't hurt that much but I think not doing would be fine as well.